agent/bench-scripts/pbench-netperf

```diff
@@ -436,23 +436,17 @@ printf "<pre>\n# these results generated with:\n# netperf %s\n\n" "$orig_cmd" >$
 printf "\n" >>$benchmark_summary_txt_file
 printf "\n" >>$benchmark_summary_html_file
 
-# disable firewall on local system
-systemctl stop firewalld
+if [[ "$postprocess_only" != "y" ]]; then
```

Comment:
We also fix this command to properly support `--postprocess-only` mode where the tools are also re-post-processed.